date-input.html

```diff
@@ -51,6 +51,7 @@
 
     <div class="horizontal layout">
       <input is="iron-input"
+          id="expMonth"
```

Comment:
No it's not.  I just added the id on the month input when I added the id to the year input. I can remove it.